openassessment/assessment/models/staff.py

```diff
@@ -0,0 +1,117 @@
+"""
+Models for managing staff assessments.
+"""
+from datetime import timedelta
+
+from django.db import models, DatabaseError
+from django.utils.timezone import now
+
+from openassessment.assessment.models.base import Assessment
+from openassessment.assessment.errors import StaffAssessmentInternalError
+
+
+class StaffWorkflow(models.Model):
+    """Internal Model for tracking Staff Assessment Workflow
```

Comment:
I think the first line should follow """, even though that's not how Google does it.